cdc/kv/regionlock/region.go

```diff
@@ -44,3 +44,27 @@ func CheckRegionsLeftCover(regions []*metapb.Region, span tablepb.Span) bool {
 	}
 	return true
```

Comment:
```suggestion
	subRegions := CutRegionsLeftCoverSpan(regions, span)
	return len(subRegions) == len(regions)
```
It is better to reuse code and tests.